packages/react-components/react-shared-contexts/src/index.ts

```diff
@@ -28,3 +28,6 @@ export {
   useCustomStyleHook as useCustomStyleHook_unstable,
 } from './CustomStyleHooksContext';
 export type { CustomStyleHooksContextValue as CustomStyleHooksContextValue_unstable } from './CustomStyleHooksContext';
+
+export { BackgroundAppearanceProvider, useBackgroundAppearance } from './BackgroundAppearanceContext';
+export type { BackgroundAppearanceContextValue } from './BackgroundAppearanceContext';
```

Comment:
Should this be exported as

```suggestion
export { BackgroundAppearanceProvider as BackgroundAppearanceProvider_unstable, useBackgroundAppearance as useBackgroundAppearance_unstable } from './BackgroundAppearanceContext';
export type { BackgroundAppearanceContextValue as BackgroundAppearanceContextValue_unstable } from './BackgroundAppearanceContext';
```

Reply:
IMO there is no more need for the `unstable` suffix anymore, our recent experience with partners is that there is full expectation for us to follow semver... @behowell  what are your thoughts on this?

Reply:
I don't think we need `_unstable` here. The suffix can be useful in cases where we expect the API to change before it is stable, but that's not the case here.

And yes, we can't break or remove any exports (including `_unstable`), so it would just add overhead to go back and eventually re-export this without the suffix, while leaving the old exports in place.